AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -2,6 +2,8 @@
 !######### CSS tweaks #########
 !##############################
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/74255
+uniqueten.net,ultraten.net#%#//scriptlet("adjust-setInterval", "wpsafe", "", "0.02")
```

Comment:
As you can see, it's a `CSS tweaks`. `JS tweaks` are located below.

Reply:
Done.